web/themes/joinup/prototype/sass/base/_ordered_list.sass

```diff
@@ -6,7 +6,7 @@
 
 article
   ol
-    padding-left: 0
+    padding-left: 15px
```

Comment:
The issue is related with bullet points which are vertically centered instead of being inline with first text line, no matter how long the text is. Do I understand it correctly?